### ISSUE TITLE
Can't view PDF documents in the browser anymore

### ISSUE THREAD

**Issue by Aster89** [qutebrowser/qutebrowser#7394]
**Version info**:

```
qutebrowser v2.5.2
Git commit: 
Backend: QtWebEngine 5.15.10, based on Chromium 87.0.4280.144
Qt: 5.15.6 (compiled 5.15.4)

CPython: 3.10.7
PyQt: 5.15.7

sip: 6.6.2
colorama: no
jinja2: 3.1.2
pygments: 2.13.0
yaml: 6.0
adblock: no
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.6
PyQt5.QtWebKitWidgets: yes
pdf.js: 2.16.105 (/usr/share/pdf.js/build/pdf.js)
sqlite: 3.39.3
QtNetwork SSL: OpenSSL 1.1.1q  5 Jul 2022

Style: QFusionStyle
Platform plugin: xcb
OpenGL: Intel, 4.6 (Compatibility Profile) Mesa 22.1.7
Platform: Linux-5.19.7-arch1-1-x86_64-with-glibc2.36, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.10/site-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /tmp/qutebrowser-basedir-w8u2jt38/cache
config: /tmp/qutebrowser-basedir-w8u2jt38/config
data: /tmp/qutebrowser-basedir-w8u2jt38/data
runtime: /tmp/qutebrowser-basedir-w8u2jt38/runtime
system data: /usr/share/qutebrowser

Autoconfig loaded: yes
Config.py: no config.py was loaded
Uptime: 0:01:21
```

**Does the bug happen if you start with `--temp-basedir`?**: yes

**Description**
If `content.pdfjs` is set `true`, the pdf never renders

**How to reproduce**

Run `:set content.pdfjs true`, and then open a PDF, such as [this](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2017/p0045r1.pdf). You should see this
![PDF](https://user-images.githubusercontent.com/20521900/189520662-4f7c9c70-9a22-48e9-871d-6500e2d07998.png)

**Comment by drehak** [comment]
I also reproduced this on Arch. Looks like this will be a problem until Qt 6 is merged (#7135, #7202).

It can temporarily be fixed by downgrading the `pdfjs` package. For me, the last available version that seems to work is 2.14.305.

**Comment by emanuele6** [comment]
This issue is a duplicate of #7335

---

@drehak
On archlinux, now, you can just install the `pdfjs-legacy` package instead of the `pdfjs` package.

Alternatively, you can use the greasemonkey userscript posted in this comment https://github.com/qutebrowser/qutebrowser/issues/7335#issuecomment-1213153686 which makes the latest `pdfjs` works just fine even using Qt5's QtWebEngine.
